farmhelper-mod/src/main/java/com/jelly/farmhelper/features/VisitorsMacro.java

```diff
@@ -153,10 +153,11 @@ public static void stopMacro() {
             LogUtils.scriptLog("Stopped visitors macro");
     }
 
-    private static boolean InJacobContest() {
+    public static boolean InJacobContest() {
```

Comment:
Tama didn't have time probably. Could you fix the method to check for "Jacob's Contest" (iirc, check in game to be sure) instead of this, because "collected" will only work without any medal